server/decorators.go

```diff
@@ -98,10 +98,15 @@ func Logger(logger *log.Logger) func(http.HandlerFunc) http.HandlerFunc {
 			p := GetPrincipal(r)
 			params := GetParams(r)
 			apiError := GetAPIError(r)
+			version := r.Header.Get("User-Agent")
+			if version == "" {
+				version = "unknown"
+			}
 			e := &reqLog{
 				Type:       "access",
 				StatusCode: 200,
 				Request:    buildRequest(r, p, params),
+				Version: 	version,
```

Comment:
Maybe just log this as UserAgent since it is possible that other things talk to the knox server over http (though not recommended) and so it is clear that this is client reported (i.e. spoof-able).